qt/widgets/common/test/BatchAlgorithmRunnerTest.h

```diff
@@ -227,7 +227,7 @@ class BatchAlgorithmRunnerTest : public CxxTest::TestSuite {
 
     TS_ASSERT_EQUALS(batchCompleteSpy.count(), 1);
     TS_ASSERT_EQUALS(batchCancelledSpy.count(), 0);
-    TS_ASSERT_EQUALS(algStartSpy.count(), 1);
+    TS_ASSERT_EQUALS(algStartSpy.count(), 2);
```

Comment:
How come the count on these signals has changed?

Reply:
Validation no longer happens when setting props, only on starting; so one extra starts but the same number completes